src/main/java/com/uid2/shared/attest/UidCoreClient.java

```diff
@@ -137,8 +128,10 @@ private HttpResponse<String> sendHttpRequest(String path, String attestationToke
         if (attestationToken != null && !attestationToken.isBlank()) {
             headers.put(Const.Attestation.AttestationTokenHeader, attestationToken);
         }
-        if (attestationJWT != null && !attestationJWT.isBlank()) {
-            headers.put(Const.Attestation.AttestationJWTHeader, attestationJWT);
+
+        String jwtToken = this.getJWT();
+        if (jwtToken != null && !jwtToken.isBlank()) {
```

Comment:
Will the JWT be blank at this point? Should we log if it is?

Reply:
It shouldn't be, but good to log it.